addons/im_livechat/static/src/legacy/widgets/feedback.js

```diff
@@ -130,11 +130,11 @@ var Feedback = Widget.extend({
                     uuid: this._livechat.getUUID(),
                     email: $email.val(),
                 }
-            }).then(function () {
-                self.$('.o_livechat_email').html($('<strong />', { text: _t('Conversation Sent') }));
-            }).guardedCatch(function () {
-                self.$('.o_livechat_email').hide();
-                self.$('.o_livechat_email_error').show();
+            }).then(() => {
+                this.$('o_livechat_email_sentLabel').show();
+            }).guardedCatch(() => {
+                this.$('o_livechat_email_receiveCopyLabel').hide();
+                this.$('o_livechat_email_receiveCopyForm').hide();
```

Comment:
The previous code replaced the whole HTML of `o_livechat_email` with "Conversation Sent".

The change here just make this visible, but keeps all the other content. Shouldn't you `.hide()` them?

(IIRC, `guardedCatch` is just when RPC fails)